src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -575,32 +575,28 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
     }
 
     renderUpgradeLinkDescription() {
```

Comment:
since we are rolling inline Notice, we should also remove this function and always show renderUpgradeInlineNotice.
showUpgradeInlineNotice can also be removed